bud.config.js

```diff
@@ -1,11 +1,8 @@
-/**
- * Bud: asset management framework.
- * @see https://roots.github.io/bud-support
- */
-const bud = require('@roots/budpack');
+// @ts-check
+const {bud} = require('@roots/bud');
```

Comment:
You'll need to use yarn link or some other solution to alias the package until it is published.